test/models/user_test.rb

```diff
@@ -0,0 +1,30 @@
+require "test_helper"
+
+describe User do
+
+  it "has a name" do
+    user = users(:bob)
+    user.name.must_equal "bobbins123"
+  end
+
+  it "requires a name" do
+    user = users(:bob)
+    user.name = ""
```

Comment:
Odd, I was able to log into the site without giving my name, so something is strange here.  Maybe you need to make sure that `user.name` must not be `nil`